caffe2/python/caffe_translator.py

```diff
@@ -234,7 +234,7 @@ def _TranslateStridePadKernelHelper(param, caffe_op):
 def TranslateConvNd(layer, pretrained_blobs, is_test):
     param = layer.convolution3d_param
     caffe_op = BaseTranslate(layer, "Conv")
-    output = caffe_op.output[0]
+    output = layer.name
```

Comment:
Do you think we will also need to change the name of these blobs when they are used as inputs to other ops?